WooCommerce/Classes/ViewRelated/Dashboard/Factories/DashboardTopBannerFactory.swift

```diff
@@ -33,8 +33,12 @@ private extension DashboardTopBannerFactory {
             NSLocalizedString("Try our improved stats", comment: "The title of the top banner on Dashboard that indicates new stats is available")
         static let info = NSLocalizedString("We’re rolling out improvements to stats for stores using the WooCommerce Admin plugin",
                                             comment: "The info of the top banner on Dashboard that indicates new stats is available")
-        // TODO-1232: update with the new `gift` icon, and draw a red dot at the top right
-        static let icon = Gridicon.iconOfType(.addImage, withSize: CGSize(width: 24, height: 24)).imageWithTintColor(StyleManager.wooGreyTextMin)
+        static let icon = Gridicon.iconOfType(.gift, withSize: CGSize(width: 24, height: 24))
```

Comment:
We are declaring all icons in UIImage+Woo. Also, to enforce at build time that the icon exist and any potential images are present, we add a test in IconTests.swift. The test id quite simple, just checking that the var in the UIImage+Woo extension does not return a nil value